src/locales/en/messages.json

```diff
@@ -3221,5 +3221,26 @@
   },
   "onlyOrgPolicyWarning": {
     "message": "Organization members who are already a member of another organization will be removed from your organization."
+  },
+  "requireSsoAuthentication": {
+    "message": "Single Sign-On Authentication"
+  },
+  "requireSsoAuthenticationPolicyDesc": {
+    "message": "Require organization members to use the Enterprise Single Sign-On method in order to log in."
+  },
+  "prerequisite": {
+    "message": "Prerequisite"
+  },
+  "requireSsoAuthenticationPolicyReq": {
+    "message": "You must enable the Single Organization Only policy before you're able to activate this policy."
+  },
+  "requireSsoAuthenticationConfigReq": {
+    "message": "Your organization's SSO configuration must be enabled before you're able to active this policy. Please visit the Business Portal to complete this action."
+  },
+  "requireSsoAuthenticationPolicyReqError": {
+    "message": "You must enable the Single Organization Only policy before you're able to activate this policy."
+  },
+  "requireSsoAuthenticationConfigReqError": {
+    "message": "Your organization's SSO configuration must be enabled before you're able to active this policy. Please visit the Business Portal to complete this action."
```

Comment:
Small typo - Please change active to **activate**
Your organization's SSO configuration must be enabled before you're able to **activate** this policy.